pom.xml

```diff
@@ -330,6 +330,27 @@ under the License.
 				</executions>
 			</plugin>
 
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>2.17</version>
+				<executions>
+					<execution>
+						<id>test</id>
+						<phase>test</phase>
+						<configuration>
+							<configLocation>style/checkstyle.xml</configLocation>
```

Comment:
The folder name "style" doesn't look quite right to me. Actually I think that unless there are other files than this file, we should just put it in the root folder.